packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -99,8 +105,10 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
           styles.picker,
           styles.pickerIsOpened,
           styles.pickerIsFocused,
-          isMonthPickerVisible && isDayPickerVisible && ('is-monthPickerVisible ' + styles.pickerIsMonthPickerVisible),
-          isMonthPickerVisible && !isDayPickerVisible && ('is-onlymonthPickerVisible ' + styles.pickerOnlyMonthPickerVisible)
+          isMonthPickerVisible && isDayPickerVisible && !showMonthPickerAsOverlay && ('is-monthPickerVisible ' + styles.pickerIsMonthPickerVisible),
+          isMonthPickerVisible && !isDayPickerVisible && !showMonthPickerAsOverlay && ('is-onlymonthPickerVisible ' + styles.pickerOnlyMonthPickerVisible),
+          isMonthPickerVisible && showMonthPickerAsOverlay && ('is-onlymonthPickerVisible ' + styles.pickerIsMonthPickerOverlayVisible),
+          isDayPickerVisible && showMonthPickerAsOverlay && styles.pickerIsDayPickerOverlayVisible
```

Comment:
I will look into this.  The main issue was that I was originally keeping the new overlay styles completely separate from the other css, but since that isn't necessary and I will be consolidating, that may fix the issue.